kolibri/core/assets/src/mixins/commonCoreStrings.js

```diff
@@ -353,7 +353,121 @@ export const coreStrings = createTranslator('CommonCoreStrings', {
   readActivity: 'Read',
   watchActivity: 'Watch',
   exploreActivity: 'Explore',
-  topicLabel: 'Topic',
+
+  folder: 'Folder',
+
+  // Library Categories
+  school: 'School',
+  basicSkills: 'Basic skills',
+  work: 'Work',
+  dailyLife: 'Daily life',
+  forTeachers: 'For teachers',
+
+  // School Categories
+  mathematics: 'Mathematics',
+  sciences: 'Sciences',
+  literature: 'Literature',
+  socialSciences: 'Social Sciences',
```

Comment:
Switch to `Social sciences` to be consistent with the sentence case in all the other categories.